alf/algorithms/mcts_algorithm.py

```diff
@@ -232,6 +244,18 @@ def __init__(
                 shape as ``steps``.
             reward_spec (TensorSpec): a rank-1 or rank-0 tensor spec representing
                 the reward(s).
+            expand_all_children (bool): If True, when a new leaf is selected, immediately
+                expand all its children. With this option, the visit
+                count does not truly refect the quality of a node. Hence it
+                should be used with (act/learn)_with_exploration_policy=True
+            expand_all_root_children (bool): whether to expand all root children
+                before search. This is described in Appendix A of "Learning and
+                Planning in Complex Action Spaces". However, our implementation is
+                different from the paper's. The paper initializes Q(s, a) for
+                root s for all the action being sampled. We expand all sampled
+                action for root s. With this option, the visit
+                count does not truly refect the quality of a node. Hence it
+                should be used with (act/learn)_with_exploration_policy=True.
```

Comment:
It is not clear to me why `act_with_exploration_policy` alleviate the problem of "visit count being off". Should this be lifted to the "There are several differences:" above and elaborate a bit?